docker-images/kafka/Dockerfile

```diff
@@ -37,6 +37,13 @@ RUN curl -LO https://github.com/danielqsj/kafka_exporter/releases/download/v${KA
 
 COPY ./exporter-scripts $KAFKA_EXPORTER_HOME
 
+#####
+# Add Cruise Control
+#####
+ENV CRUISE_CONTROL_HOME=/opt/cruise-control
+RUN mkdir $CRUISE_CONTROL_HOME
+COPY ./cruise-control-scripts $CRUISE_CONTROL_HOME
```

Comment:
Thsi section is called _Add Cruise Control_ But it seems Cruise Control is actually added on line 60. Any reason why not move it here? Or we should at least rename this.